docs/application/native/guides/messaging/push-server.md

```diff
@@ -52,30 +52,29 @@ To send push notifications:
 
      ```
      {
-         "messages":
-         [{
-             /* Other content */
-             "message": "action=ALERT&badgeOption=INCREASE&badgeNumber=1&alertMessage=Hi",
-             "appData": "{id:asdf&passwd:1234}",
-             /* Other content */
-         ]}
+         /* Other content */
+         "message": "action=ALERT&badgeOption=INCREASE&badgeNumber=1&alertMessage=Hi",
+         "appData": "{id:asdf&passwd:1234}",
+         /* Other content */
      }
      ```
 
-   - If you have data to send to the application but no need to notify the user, use the action field on the same level as the messages field, instead of within the messages field, and do not include the message field itself. In this case, the notification is delivered with the best effort.
+   - If you have data to send to the application but no need to notify the user right away, use the action field on the same level as the message field, instead of within the message field, and do not include the message field itself. In this case, the notification is delivered with the best effort.
```

Comment:
If you have to send data to the application, and if the user need not be notified immediately, use the action field at the same level as the `message` field, and do not include the `message` field. In this case, the notifications is delivered with the best effort.

Reply:
Hello, @annie-samsung. Would you please recheck this reviewed sentence? It sounds a little bit lengthy to me, and I think tidy sentence would be better for "Developer document".

Reply:
Hello @mijin85cho,

How about re-wording it as ( I am removing this bit: "instead of within the message field" as we are anyways recommending not include the message field itself):

If you have to send data to an application without notifying the user immediately, use the action field. The action field must be used at the same level as message field, and do not include the message field itself. In this case, the notification is delivered on best-effort basis.

Reply:
Hello, @annie-samsung 
Thanks for the rewriting. I think this sentence is good.

So, @titicoco,
would you please update this PR using the suggested sentence?
I hope @divyk can check the latest version after your update and approve it when there is no issue.